source/Halibut/ServiceModel/PendingRequestQueueAsync.cs

```diff
@@ -16,22 +16,22 @@ public class PendingRequestQueueAsync : IPendingRequestQueue
         readonly SemaphoreSlim queueLock = new(1, 1);
         readonly AsyncManualResetEvent itemAddedToQueue = new(false);
         readonly ILog log;
-        readonly TimeSpan pollingQueueWaitTimeout;
+        readonly HalibutTimeoutsAndLimits halibutTimeoutsAndLimits;
```

Comment:
Lol, I was undecided on whether to do that or not.

I went with what I did so that if we need more settings from `HalibutTimeoutsAndLimits` in future, it would be much easier to add.

But I'm happy with passing the extra bool in specifically, as I was 50/50 on it 😁

Reply:
@LukeButters , [this commit](https://github.com/OctopusDeploy/Halibut/pull/577/commits/3cafbe1a3d165d6536eef56f8d6229c665e67638) addresses this.

Reply:
lgtm